sklearn/linear_model/logistic.py

```diff
@@ -1213,6 +1217,10 @@ def fit(self, X, y, sample_weight=None):
                              self.dual)
 
         if self.solver == 'liblinear':
+            if self.n_jobs != -1:
+                warnings.warn(
```

Comment:
I think you can use a single `if` statement. I think that you should use quote and not use "You're" :)
A possible solution (feel free to edit).

```python
if self.solver == 'liblinear' and self.n_jobs != 1:
    warnings.warn("With 'solver'='liblinear', 'n_jobs' != 1"
                  "has no effect. Got 'n_jobs='{}.".format(self.n_jobs)))
```